### PR TITLE
US 1583733 Add missing language IDs - Part2

### PR DESCRIPTION
**Pull request by v-thpra**
[User Story 1583733](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1583733): Content Update: add missing language ID

Per the [CATS Report from 11-11-2019](https://contentqacats.azurewebsites.net/CheckResults/2019111100008/Report/) there are 98 code blocks without language IDs:
- 48 of them were reviewed previously and it was decided that they should remain blank.
- 50 of them are filled in by the changes in this PR. (several instances are cleared by changes in two include files)

### CODE REVIEW

**Comment by v-thpra** [comment]
@mairaw I'm assuming that the 9 reviewers are because I edited 2 include files in the Retargeting Changes for Migration from .NET Framework area? My change in each file was simple - I added a "class=language" value to a <code> tag that was missing one. These accounted for 34 of the missing ID's.

File: docs/standard/exceptions/exception-class-and-properties.md [hunk 1]
```diff
@@ -13,12 +13,10 @@ ms.author: "mairaw"
 
 The <xref:System.Exception> class is the base class from which exceptions inherit. For example, the <xref:System.InvalidCastException> class hierarchy is as follows:
 
-```
-Object
-  Exception
-    SystemException
-       InvalidCastException
-```
+Object
```
Comment (Youssef1313) [inline docs/standard/exceptions/exception-class-and-properties.md]:
```suggestion
Object\
```

File: docs/standard/exceptions/exception-class-and-properties.md [hunk 2]
```diff
@@ -13,12 +13,10 @@ ms.author: "mairaw"
...
 The <xref:System.Exception> class is the base class from which exceptions inherit. For example, the <xref:System.InvalidCastException> class hierarchy is as follows:
 
-```
-Object
-  Exception
-    SystemException
-       InvalidCastException
-```
+Object  
+&nbsp;&nbsp;Exception
```
Comment (Youssef1313) [inline docs/standard/exceptions/exception-class-and-properties.md]:
```suggestion
&nbsp;&nbsp;Exception\
```

File: docs/standard/exceptions/exception-class-and-properties.md [hunk 3]
```diff
@@ -13,12 +13,10 @@ ms.author: "mairaw"
...
 
-```
-Object
-  Exception
-    SystemException
-       InvalidCastException
-```
+Object  
+&nbsp;&nbsp;Exception  
+&nbsp;&nbsp;&nbsp;&nbsp;SystemException
```
Comment (Youssef1313) [inline docs/standard/exceptions/exception-class-and-properties.md]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;SystemException\
```

File: docs/standard/exceptions/exception-class-and-properties.md [hunk 4]
```diff
@@ -13,12 +13,10 @@ ms.author: "mairaw"
...
-```
-Object
-  Exception
-    SystemException
-       InvalidCastException
-```
+Object  
+&nbsp;&nbsp;Exception  
+&nbsp;&nbsp;&nbsp;&nbsp;SystemException  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;InvalidCastException
```
Comment (Youssef1313) [inline docs/standard/exceptions/exception-class-and-properties.md]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;InvalidCastException\
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
Related to #15689

File: docs/standard/exceptions/exception-class-and-properties.md (hunk 1 above)
Comment (tdykstra) [reply docs/standard/exceptions/exception-class-and-properties.md]: @Youssef1313 Why are you suggesting a backslash at the end of these lines?

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/standard/exceptions/exception-class-and-properties.md (hunk 1 above)
Comment (Youssef1313) [reply docs/standard/exceptions/exception-class-and-properties.md]: For some reason, it won't render correctly in languages other than English, see #15689.

File: docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md [hunk 5]
```diff
@@ -202,13 +202,11 @@ dotnet sln add src/TraderSys.PortfolioData
...
 Once the library has been created and added to the solution, delete the generated `Class1.cs` file and copy the files from the WCF solution's library into the new class library's folder, keeping the folder structure.
 
-```
-Models
-  Portfolio.cs
-  PortfolioItem.cs
-IPortfolioRepository.cs
-PortfolioRepository.cs
-```
+- Models
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md]: I'd keep this one as a code block with no identifier

File: docs/spark/how-to-guides/debug.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ spark-submit \
 
 When you run the command, you see the following output:
 
-```
+```output
```
Comment (mairaw) [inline docs/spark/how-to-guides/debug.md]:
```suggestion
```console
```

File: docs/standard/assembly/names.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ An assembly's name is stored in metadata and has a significant impact on the ass
 
  The runtime uses this information to locate the assembly and differentiate it from other assemblies with the same name. For example, a strong-named assembly called `myTypes` could have the following fully qualified name:
 
-```
+```console
```
Comment (mairaw) [inline docs/standard/assembly/names.md]: I'd keep these with no id

File: docs/standard/exceptions/exception-class-and-properties.md (hunk 1 above)
Comment (mairaw) [reply docs/standard/exceptions/exception-class-and-properties.md]: We could also make these xref links like
```suggestion
<xref:System.Object>\
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Nice changes @v-thpra. I've left you a few comments for you to consider.

File: docs/standard/exceptions/exception-class-and-properties.md (hunk 1 above)
Comment (Youssef1313) [reply docs/standard/exceptions/exception-class-and-properties.md]: @mairaw, You suggested to keep it in a code block without an id, but I think xrefs won't work inside code blocks

File: docs/standard/assembly/names.md (hunk 7 above)
Comment (v-thpra) [reply docs/standard/assembly/names.md]: Made this change.

File: docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md (hunk 5 above)
Comment (v-thpra) [reply docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md]: Made this change.

**Comment by v-thpra** [comment]
@mairaw @Youssef1313 I used the suggestion from Maira and made these xref links, but I also kept the "&nbsp" to show the heirarchy in hopes we could drop the code block.

File: docs/standard/exceptions/exception-class-and-properties.md (hunk 1 above)
Comment (v-thpra) [reply docs/standard/exceptions/exception-class-and-properties.md]: See my comment below. I tried the xref suggestion.

**Comment by Youssef1313** [comment]
@v-thpra, This is okay to use &nbsp; to show the hierarchy, but I'd still use a backslash instead of the two spaces.

**Comment by v-thpra** [comment]
@Youssef1313 Thanks, good suggestion! I added the backslashes and it looks good in preview.

**Comment by Youssef1313** [comment]
@v-thpra, hope it looks good in the localized versions as well. 😄

File: docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md [hunk 8]
```diff
@@ -203,9 +203,9 @@ dotnet sln add src/TraderSys.PortfolioData
 Once the library has been created and added to the solution, delete the generated `Class1.cs` file and copy the files from the WCF solution's library into the new class library's folder, keeping the folder structure.
 
 ```
-Models
-  Portfolio.cs
-  PortfolioItem.cs
+Models
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md]:
```suggestion
Models
```

File: docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md [hunk 9]
```diff
@@ -203,9 +203,9 @@ dotnet sln add src/TraderSys.PortfolioData
 Once the library has been created and added to the solution, delete the generated `Class1.cs` file and copy the files from the WCF solution's library into the new class library's folder, keeping the folder structure.
 
 ```
-Models
-  Portfolio.cs
-  PortfolioItem.cs
+Models  
+    Portfolio.cs
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md]:
```suggestion
  Portfolio.cs
```

File: docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md [hunk 10]
```diff
@@ -203,9 +203,9 @@ dotnet sln add src/TraderSys.PortfolioData
...
+    PortfolioItem.cs
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/migrate-request-reply.md]:
```suggestion
  PortfolioItem.cs
```

File: docs/spark/how-to-guides/debug.md [hunk 11]
```diff
@@ -24,7 +24,7 @@ spark-submit \
 
 When you run the command, you see the following output:
 
-```
+```console
 ***********************************************************************
 * .NET Backend running debug mode. Press enter to exit *
```
Comment (mairaw) [inline docs/spark/how-to-guides/debug.md]: @mamccrea unrelated to this PR, but do you know where this message could be updated? back end should be two words.

Comment (Youssef1313) [reply docs/spark/how-to-guides/debug.md]: @mairaw, I think it's [here](https://github.com/dotnet/spark/blob/master/src/scala/microsoft-spark-3.0.x/src/main/scala/org/apache/spark/deploy/dotnet/DotnetRunner.scala#L150)

Comment (mairaw) [reply docs/spark/how-to-guides/debug.md]: Thanks @Youssef1313! I've opened a PR there.

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
Thank you @v-thpra!

**Pull request merged**